addons/sale_pdf_quote_builder/wizards/res_config_settings.py

```diff
@@ -7,4 +7,6 @@ class ResConfigSettings(models.TransientModel):
     _inherit = 'res.config.settings'
 
     sale_header = fields.Binary(related='company_id.sale_header', readonly=False)
+    sale_header_name = fields.Char(related='company_id.sale_header_name')
     sale_footer = fields.Binary(related='company_id.sale_footer', readonly=False)
+    sale_footer_name = fields.Char(related='company_id.sale_footer_name')
```

Comment:
Should be hopefully removed later when we have a cleaner way to display the filename.